src/app/personside/dialogpanel/component/Temavelger.tsx

```diff
@@ -32,23 +40,24 @@ function Temavelger(props: Props) {
         props.setTema(tema);
     };
     return (
-        <Select
-            // @ts-ignore
-            selectRef={ref => (selectRef = ref)}
-            label="Tema"
-            onChange={velgTemaHandler}
-            feil={props.visFeilmelding ? { feilmelding: 'Du må velge tema' } : undefined}
-            value={props.tema ? props.tema.kodeRef : ''}
-        >
-            <option value="" disabled>
-                Velg tema
-            </option>
-            {temaValg.map(valg => (
-                <option key={valg.kodeRef} value={valg.kodeRef}>
-                    {valg.beskrivelse}
+        <SkjemaGruppe feil={props.visFeilmelding ? { feilmelding: 'Du må velge tema' } : undefined}>
+            <StyledSelect
+                // @ts-ignore
+                selectRef={ref => (selectRef = ref)}
```

Comment:
Denne syntaksen er vel utdatert?

Reply:
Ikke for Select fra nav-frontend :-P Derfor jeg måtte klinke til med en // @ts-ignore

Reply:
aha.. :(